CMSgov/easi-app
# [EASI-3802] Advice letter RTE bug fix

# EASI-3802
## Changes and Description

This fixes the first bug described in the ticket where HTML tags are showing up in the advice letter review step. I'm waiting for clarification on the second bug, so that will be handled in a separate PR.

- Added `RichTextViewer` to notes accordion in advice letter review step



## How to test this change

- Start advice letter for TRB request
- Create admin note about advice letter
- Go to advice letter final review step (/trb/:trbRequestId/advice/review)
- HTML tags should not show in note



## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
